docs/modules/operation/pages/kafka-producer/enable-kafka.adoc

```diff
@@ -6,20 +6,27 @@ The Kafka Producer is disabled by default and can be enabled as follows.
 First, log in to the Karaf shell of your {page-component-title} instance and configure the Kafka client settings to point to your Kafka broker.
 See link:https://kafka.apache.org/10/documentation.html#producerconfigs[Producer Configs] for a complete list of available options.
 
+.Connect to Karaf shell
 [source, console]
 ----
-$ ssh -p 8101 admin@localhost
-...
-admin@opennms()> config:edit org.opennms.features.kafka.producer.client
-admin@opennms()> config:property-set bootstrap.servers 127.0.0.1:9092
-admin@opennms()> config:update
+ssh -p 8101 admin@localhost
 ----
 
+.Configure the Kafka producer client
+[source, karaf]
+----
+config:edit org.opennms.features.kafka.producer.client
+config:property-set bootstrap.servers kafka-broker-ip:9092 <1>
+config:update
+----
+<1> Set the Kafka broker IP by replacing the kafka-broker-ip and port you want to connect.
```

Comment:
```suggestion
<1> Set the Kafka broker IP by replacing the kafka-broker-ip and port you want to connect to.
```

Reply:
Is a preposition word we want to end a sentence with?

Reply:
It's not usually good practice, but in this situation it's okay. Changing it to "with which you want to connect" would be too awkward. Most style guides allow it now under these types of circumstances. 
We could say "... and connection port." if that would still make sense to users.

Reply:
Cool cool cool